pygluu/kubernetes/gui/views/wizard.py

```diff
@@ -824,8 +842,6 @@ def couchbase():
         data["COUCHBASE_NAMESPACE"] = form.couchbase_namespace.data
         data["COUCHBASE_CLUSTER_NAME"] = form.couchbase_cluster_name.data
         data["COUCHBASE_URL"] = form.couchbase_url.data
-        data["COUCHBASE_SUPERUSER"] = form.couchbase_superuser.data
```

Comment:
why is this being removed ?

Reply:
Ah sorry, i think the ide not refreshed when i checkout new branch..